### PR TITLE
Add the pelotech/drone-helm3 plugin

### PR DESCRIPTION
**Pull request by ErinCall**

### CODE REVIEW

**Comment by ErinCall** [comment]
cc @tboerger

**Comment by josmo** [comment]
@tboerger looks like there's two drone-helm3 plugins now (just saw the other from @foosinn) I started an issue on https://github.com/bitsbeats/drone-helm3/issues/1 to see about merging them since they have a lot over overlap. I think this one might be a tiny bit further along right now feature/doc wise but I don't want to step on toes :)

File: content/pelotech/drone-helm3/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+version: '0.8'
```
Comment (tboerger) [inline content/pelotech/drone-helm3/index.md]: This attribute got to be dropped

**Comment by tboerger** [comment]
It's totally valid if there are multiple plugins that provide nearly the same functionality. It's up to the authors to team up or not. The plugin index should list both as long as they exist.

**Comment by josmo** [comment]
@tboerger cool cool :)

**Pull request merged**